File: ClientApp/models/LastOpenedDocument.ts
import { Template } from "./Template"

export class LastOpenedDocument {
    lastOpenedDocumentId: number;
    patient: string;
    lastOpenedTime: Date;
    templateId: number;

    //not working and I don't now why
    // public constructor() {
    //     this.lastOpenedTime = new Date(this.lastOpenedTime);
    //     alert(typeof(this.lastOpenedTime));
    // }
}

File: Models/LastOpenedDocument.cs
using System;
using Newtonsoft.Json;

namespace MedHelp.Models
{
    public class LastOpenedDocument
    {
        public int LastOpenedDocumentId { get; set; }
        public string Patient { get; set; }
        public DateTime LastOpenedTime { get; set; }
        public int TemplateId { get; set; }
        public FormModel FormModel{ get; set; }
        public int FormModelId { get; set; }
        public string UserId { get; set; }
        [JsonIgnore]
        public Template Template { get; set; }
    }
}